include/api.php

```diff
@@ -2449,7 +2449,8 @@ function api_convert_item($item)
 	}
 
 	$statushtml = BBCode::convert(api_clean_attachments($body), false);
-
+	$renderedhtml = $item['rendered-html'];
```

Comment:
You possibly should use ```BBCode::convert($body, false);``` instead. The ```false``` is disabling OEmbed. OEmbed can create inline frames or some Javascript foo.

Reply:
But then it is the same limited output as with $statushtml, isn't it? I was using the rendered-html from database, because the field already contains extended html data, most probably previously retrieved from Oembed etc. And doing again a conversion to html code which is already available would only cause unneeded workload, right?

Reply:
The workload isn't a real problem. The problem is that that HTML would possibly contain ```<iframe>``` elements - which would be problematic.